### PR TITLE
Run Python tests

### PR DESCRIPTION
**Pull request by shusso**
start_script is now divided to its own Python related file

### CODE REVIEW

File: test_start_helpers.sh [hunk 1]
```diff
@@ -92,6 +92,25 @@ function get_ios_device_name {
   fi
 }
 
+
+function start_script_py {
```
Comment (ujappelbe) [inline test_start_helpers.sh:98]: Should we split up start_script-methods so we have:

```
function install_and_start_script {
  install_libs
  start_script
}
```

Both for python and js. This code looks good but I am worried it's going to end up like the huge function for the js start_script when special cases starts piling up.

**Comment by shusso** [comment]
@antonsi @ujappelbe please check through :)

File: install_and_start_py.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        JUNIT_REPORT_NAME=${JUNIT_REPORT_NAME:="No Device Name Set"}
+        export JUNIT_REPORT_NAME
+    fi
+}
+
+function start_tests_py {
+    echo "Running tests '$TEST'"
+    python "$TEST"
+    return $?
+}
```
Comment (antnsi) [inline install_and_start_py.sh]: missing eof plz fix

**Comment by antnsi** [comment]
there is missing eof, ok for the rest

File: install_and_start_py.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    echo "installing py libs"
+    if [ "$TESTDROID" == "1" ]; then
+        sudo easy_install pip
+        sudo pip install Appium-Python-Client unittest-xml-reporting
+        #sudo brew install ios-webkit-debug-proxy
+
+        export JUNIT_REPORT_STACK=1
+        export JUNIT_REPORT_PATH="TEST-all.xml"
+        JUNIT_REPORT_NAME="$DEVICE_NAME"
+        JUNIT_REPORT_NAME=${JUNIT_REPORT_NAME:="No Device Name Set"}
```
Comment (ujappelbe) [inline install_and_start_py.sh:13]: I recall some system, (maybe testdroid or jenkins) has issues with junit.xml parsing if the root name contains special characters or whitespaces. 

If there are issues then try filtering out all non-alphanumerals/whitespaces.

**Pull request merged**